<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="lastPlaylistCheckBox">
        <property name="toolTip">
         <string>When Jupii connects to a device, the last playlist will be automatically loaded.</string>
        </property>
        <property name="whatsThis">
         <string>When Jupii connects to a device, the last playlist will be automatically loaded.</string>
        </property>
        <property name="text">
         <string>Start with last playlist</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="netInfsLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Network interface</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="netiInfsComboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Experimental</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="imageCheckBox">
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string/>
           </property>
           <property name="text">
            <string>Image content</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="pulseCheckBox">
           <property name="text">
            <string>Capture audio output</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="pulseModeLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Audio output format</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="pulseModeComboBox">
             <item>
              <property name="text">
               <string>PCM 44100Hz stereo (default)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>PCM 44100Hz mono</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>PCM 22050Hz stereo</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>PCM 22050Hz mono</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="allDevicesCheckBox">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>All devices visible</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="remoteContentModeLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Internet streaming mode</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="remoteContentModeComboBox">
             <item>
              <property name="text">
               <string>Proxy (default)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Redirection</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
